### PR TITLE
Magento Devdocs: Fixed Typo and Formatted Code

### PR DESCRIPTION
**Pull request by vishal-7037**
## Purpose of this PR

Properly formatted the code.

## Affected URLs

- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html

## Links to source code

- https://prnt.sc/nde7nz

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4285) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4285) before we can accept your contribution.<br/>**4** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: mage2pratik<br/>:white_check_mark: keharper<br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: vishal-7037<br/>:x: AlexMaxHorkun<br/>:x: Vishal Sutariya<hr/>**Vishal Sutariya** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/send-request.md [hunk 1]
```diff
@@ -3,7 +3,25 @@ group: graphql
 title: GraphQL requests
 ---
 
-In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
+Magento GraphQL supports the HTTP GET and POST methods. You can send a query as a GET or POST request. GET requests can be cached, although Magento does not currently support this feature. Mutations must be POST requests.
```
Comment (rogyar) [inline guides/v2.3/graphql/send-request.md]: Please note, Magento does not have GET requests support within a scope of GraphQl API. This feature is merged and planned to be delivered in the future releases.

**Comment by jeff-matthews** [comment]
@vishal-7037, why are there so many commits attached to this PR? Your local branch must be behind master. Did you cut your branch off of `develop` by any chance?

**Comment by keharper** [comment]
@vishal-7037  It appears that you have gone through the `develop` branch, found some recent changes, and applied them to the `master` branch. The `develop` branch contains documentation for unreleased code--2.3.2 in this case. The `master` branch is for released code (2.3.1 and older). 

Please back out all changes, except for those in the `db-schema.md` file. All the other changes are specific to the 2.3.2 release, and therefore don't belong in `master`.

**Pull request closed without merging**